src/simperium/channel.js

```diff
@@ -12,68 +108,99 @@ const CODE_INVALID_VERSION = 405;
 const CODE_EMPTY_RESPONSE = 412;
 const CODE_INVALID_DIFF = 440;
 
-var operation = {
+type Modify = 'M';
+type Remove = '-';
+type OperationType = Modify | Remove;
```

Comment:
for funnies check out the [utility types](https://flow.org/en/docs/types/utilities/)

```flow
const operations = {
  Modify: 'M',
  Remove: '-'
};

type Operation = $Values<typeof operations>;
```

something like that